src/gadfly.js

```diff
@@ -177,14 +177,14 @@ Gadfly.helpscreen_visible = function(event) {
     helpscreen_visible(this.plotroot());
 };
 var helpscreen_visible = function(root) {
-    root.select(".helpscreen").animate({opacity: 1.0}, 250);
+    root.select(".helpscreen").animate({"fill-opacity": 1.0}, 250);
```

Comment:
this appears to be a duplicate of https://github.com/GiovineItalia/Gadfly.jl/commit/c62ef81c7dc792792f0e757f5daf657f666de68d, and is how the code already looks on master.  how is it then that this is showing up as a change in the diff?

Reply:
At some point I rebased this branch on master, because I started it before c62ef81. What does github think the diff is relative to?

Reply:
if you click on the "commits" tab above it says "bjarthur authored and Mattriks committed 12 days ago".  i think the rebase messed things up.  am worried master will get messed up if i merge this.  can you please see if you can fix it?

Reply:
how did you do your rebase?  my workflow is as follows:

```
git checkout master
git pull
git checkout -b newPR
git add
git commit
git checkout master
git pull
git checkout newPR
git rebase master
```

the key here is that master is rebase onto newPR, not vice versa, which is what i think you might have done.